mlflow/models/utils.py

```diff
@@ -178,14 +198,16 @@ def _read_example(mlflow_model: Model, path: str):
     if mlflow_model.saved_input_example_info is None:
         return None
     example_type = mlflow_model.saved_input_example_info["type"]
-    if example_type not in ["dataframe", "ndarray"]:
+    if example_type not in ["dataframe", "ndarray", "csc", "csr"]:
```

Comment:
What do you think about the example type being more verbose? e.g. something like `sparse_matrix_csc`?

Not a big deal, but I wonder if it would make the metadata more readable.